apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -39,6 +40,12 @@
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
         </dependency>
+        <dependency>
+            <!-- Templating library -->
+            <groupId>com.github.jknack</groupId>
+            <artifactId>handlebars</artifactId>
+            <version>${handlebars.version}</version>
+        </dependency>
```

Comment:
Has this gone through the process for new dependencies? I think last time it was requested that even libraries for the RDA bridge should land in a document on confluence.

Reply:
We have not.  This was my fault, I asked Kyle to find a templating library thinking that it be ok to include since this isn't going to run in AWS.  For our needs, I think we can do something much simpler using formatting strings.  @KyRobbins apologies for leading you down the wrong path, but what if we just use a syntax like `('"%s",',10)` to indicated the format for each value and the max number?

Reply:
after talking about it, we decided to implement just a simple string format for now to do the bulk of the work and do manual edits afterwards to finalize.